src/pudl/etl/__init__.py

```diff
@@ -39,6 +39,7 @@
     *load_assets_from_modules([pudl.transform.ferc1], group_name="norm_ferc1"),
     *load_assets_from_modules([pudl.extract.ferc714], group_name="raw_ferc714"),
     *load_assets_from_modules([pudl.transform.ferc714], group_name="clean_ferc714"),
+    *load_assets_from_modules([pudl.output.ferc714], group_name="norm_ferc714"),
```

Comment:
I'm a little confused by what assets live in `norm_ferc714`. It looks like a mix of intermediate, 861, and 714 and "normalized" assets. Ideally, we'd have separate groups for intermediate 714 assets, intermediate 861 assets, normalized 714 assets and normalized 861 assets. I can't think of a great place to place the logic for these 4 groups given our current structure. Once we iron out our naming conventions and code structure #2517, we'll probably have a module and asset group for each of the four groups listed above.

I think leaving this collection of assets in `pudl.output.ferc714` is ok for now. Can you think of another way to structure these asset groups?

Reply:
I agree, this isn't actually normalized but its own analysis layer building on 861/714 tables. Perhaps I could call it `respondents` to differentiate it from the ETL assets? Or leave it as is until the reorganization?

Reply:
Yeah maybe leave everything in `pudl.output.ferc714` for now and rename the group to `respondents`? Does it make sense to include the data source in the asset group for now? `respondents_ferc714`, `service_territory_eia861`, `state_demand_ferc714`.

Reply:
Done!